safe-global/web-core
# fix: Detect replacement tx when creating a safe

## What it solves

Calling `provider.waitForTransaction` with the cached `txHash` will not detect if the transaction was cancelled or sped-up. Instead we now call `provider._waitForTransaction` with a `replacement` object and handle those cases.

## How to test

1. Create a new safe
2. Reload the page while the tx is executing
3. Cancel or speed-up the transaction in your wallet
4. Observe the Status changing to ERROR or SUCCESS
